elements/pl-order-blocks/pl-order-blocks.py

```diff
@@ -318,6 +322,9 @@ def parse(element_html, data):
     if answer_raw_name in data['raw_submitted_answers']:
         student_answer = data['raw_submitted_answers'][answer_raw_name]
 
+    if student_answer is None:
+        student_answer = '[]'
```

Comment:
Nit: I would simplify this and the preceding lines to something like `student_answer = data['raw_submitted_answers'].get(answer_raw_name, '[]')`.

Reply:
oh, duh